packages/cli/README.md

```diff
@@ -306,6 +306,31 @@ COMMON FLAGS
 
 DESCRIPTION
   get a specific app or a list of apps
+
+EXAMPLES
+  list all apps
+
+    $ smartthings apps
+
+  list the first app in the list retrieved by running "smartthings apps"
+
+    $ smartthings apps 1
+
+  list an app by id
+
+    $ smartthings apps <app-id>
+
+  include URLs and ARNs in the output
+
+    $ smartthings apps --verbose
+
+  list only SERVICE classification devices
+
+    $ smartthings apps --classification SERVICE
+
+  list only API only apps
+
+    $ smartthings apps --type API_ONLY
```

Comment:
It looks like I accidentally committed these examples a couple weeks ago with the table-summary pull request. We still need to review them but they're good examples so I'll leave them for now.

https://github.com/SmartThingsCommunity/smartthings-cli/pull/426